src/convertToObject.js

```diff
@@ -12,7 +12,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((acc, cur) => {
+    const [key, value] = cur.split(':');
+
+    if (key && value) {
```

Comment:
this will also work if my key is '   ' and my value is '     '; this expression will only equal to false if one of two strings is an empty string. You have not covered the case where both strings contain only spaces